site/src/i18n/en/workspacePage.json

```diff
@@ -0,0 +1,7 @@
+{
+  "workspace_schedule_button": {
+    "schedule": "Schedule",
+    "edit_deadline_minus": "Subtract one hour",
```

Comment:
is snake case a convention for i18n?

Reply:
Oh no, it's not. We can change if you'd like - it's just a convention I've used in the past because I find it a bit easier to read/recognize the translation keys. I don't mind changing it back to camel if you'd prefer - let me know!

Reply:
I'm used to camel case but whatever the team prefers!

Reply:
I'll switch it and we can always pivot if we want

Reply:
My vote would be for camel as well.